### PR TITLE
Add Organization endpoints

### PR DESCRIPTION
**Pull request by gssbzn**

### CODE REVIEW

File: cloudmanager/organizations.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+// OrganizationsService is an interface for interfacing with the Projects
+// endpoints of the MongoDB Atlas API.
+// See more: https://docs.cloudmanager.mongodb.com/reference/api/organizations/
+type OrganizationsService interface {
+	GetAllOrganizations(context.Context) (*Organizations, *atlas.Response, error)
+	GetOneOrganization(context.Context, string) (*Organization, *atlas.Response, error)
+	GetProjects(context.Context, string) (*Projects, *atlas.Response, error)
+}
+
+// OrganizationsServiceOp handles communication with the Projects related methos of the
```
Comment (savannaendicott) [inline cloudmanager/organizations.go]: super nit : methods

File: cloudmanager/organizations.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+// Organizations represents a array of organization
+type Organizations struct {
+	Links      []*atlas.Link   `json:"links"`
+	Results    []*Organization `json:"results"`
+	TotalCount int             `json:"totalCount"`
+}
+
+// GetAllOrganizations gets all project.
```
Comment (savannaendicott) [inline cloudmanager/organizations.go]: gets all organizations

File: cloudmanager/organizations.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+	}
+
+	if l := root.Links; l != nil {
+		resp.Links = l
+	}
+
+	return root, resp, nil
+}
+
+// GetOneOrganization gets a single project.
```
Comment (savannaendicott) [inline cloudmanager/organizations.go]: gets a single organization

File: cloudmanager/organizations.go [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+	root := new(Organization)
+	resp, err := s.client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, err
+}
+
+// GetProjects gets a single project by its name.
```
Comment (savannaendicott) [inline cloudmanager/organizations.go]: GetProjects gets all projects in an org by orgId

**Review by savannaendicott — APPROVED** [review]
a couple of nitpicks for method descriptions, otherwise LGTM! 👍

File: cloudmanager/organizations.go (hunk 1 above)
Comment (gssbzn) [reply cloudmanager/organizations.go]: 🤔 weird, misspell should catch that

**Comment by gssbzn** [comment]
Fixed al the copy/pasta, thx for the extra 👀

**Pull request merged**